### ISSUE TITLE
Journey timers look reversed 

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-dev#6702]
This journey http://kademi.admin.kademi.us/funnels/incentive-loyalty-2019-new-leads/version1/#journeyBuilder-tab

The first timer goal in the journey is PM 
![image](https://user-images.githubusercontent.com/2201340/52597627-8fe3d500-2ea7-11e9-88b2-62bf0266eea5.png)

However the lead timer once triggered shows AM 
![image](https://user-images.githubusercontent.com/2201340/52597660-a853ef80-2ea7-11e9-937b-039b3a9e3a10.png)

**Comment by bradmac** [comment]
i think the popup text might just be showing in US time

@Uafinder can you confirm timers are firing at the correct time, allowing for account timezone?

**Comment by paddyow** [comment]
Note the lead page on the profile shows the time correctly, its only when you click through to the lead.

Lead page on profile as below:
![image](https://user-images.githubusercontent.com/2201340/52616187-c6434380-2eeb-11e9-84b4-de85544f318c.png)

**Comment by bradmac** [comment]
yep, i think its just that the timezone isnt being applied to the popup text
